### PR TITLE
Add WGE 0.22.0 release notes

### PR DESCRIPTION
**Pull request by serboctor**
Added WGE 0.22.0 release notes to the website's release page.

### CODE REVIEW

File: website/docs/releases.mdx [hunk 1]
```diff
@@ -10,6 +10,39 @@ import TierLabel from "./_components/TierLabel";
...
+#### Explorer
+
+- Explorer supports now Flux sources.
+- Applications UI and Sources UI could be configured to use Explorer backend to improve UI experience.
+
+#### GitopsSets
+
+- Now supports correctly templating numbers and object chunks
+
+#### Cluster Bootrap
```
Comment (enekofb) [inline website/docs/releases.mdx:28]:
```suggestion
#### Cluster Bootstrapping
```

File: website/versioned_docs/version-0.22.0/releases.mdx [hunk 2]
```diff
@@ -10,6 +10,39 @@ import TierLabel from "./_components/TierLabel";
...
+#### Explorer
+
+- Explorer supports now Flux sources.
+- Applications UI and Sources UI could be configured to use Explorer backend to improve UI experience.
+
+#### GitopsSets
+
+- Now supports correctly templating numbers and object chunks
+
+#### Cluster Bootrap
```
Comment (enekofb) [inline website/versioned_docs/version-0.22.0/releases.mdx:28]:
```suggestion
#### Cluster Bootstrapping
```

**Review by enekofb — APPROVED** [review]

File: website/versioned_docs/version-0.22.0/releases.mdx (hunk 2 above)
Comment (enekofb) [reply website/versioned_docs/version-0.22.0/releases.mdx:28]: just minor change in the header

**Review by foot — APPROVED** [review]
Nice!

**Pull request merged**